aggregator/src/aggregator.rs

```diff
@@ -1089,44 +1140,124 @@ macro_rules! vdaf_ops_dispatch {
                 let $verify_key = verify_key;
                 type $Vdaf = ::prio::vdaf::prio3::Prio3Histogram;
                 const $VERIFY_KEY_LENGTH: usize = ::janus_core::task::VERIFY_KEY_LENGTH;
+                $(
+                    type $DpStrategy = janus_core::dp::NoDifferentialPrivacy;
+                    let $dp_strategy = &Arc::new(janus_core::dp::NoDifferentialPrivacy {});
+                )?
                 $body
             }
 
             #[cfg(feature = "fpvec_bounded_l2")]
-            crate::aggregator::VdafOps::Prio3FixedPoint16BitBoundedL2VecSum(vdaf, verify_key) => {
+            crate::aggregator::VdafOps::Prio3FixedPoint16BitBoundedL2VecSum(vdaf, verify_key, _dp_strategy) => {
```

Comment:
Still needs to be addressed.

Reply:
Yes, the problem here is that whether this variable is used depends on the given macro parameters. If `$dp_strategy` and `$DpStrategy` are given, then it is used. Otherwise it is not. This means that changing the name to `dp_strategy` results in lots of warnings, since in most invocations of `vdaf_ops` it is, in fact, not used.

Another solution to this problem would be to add `#[allow(unused_variables)]` before the match case but that would also suppress warnings about unused variables in the body. There are also some other solutions, with their own trade-offs, but I think using the underscore notation is the most straightforward solution.

Of course I'm open to other suggestions, in the meantime I'm going to add a short comment explaining the naming which I think should solve the situation for the most part.